Code/GraphMol/ForceFieldHelpers/CrystalFF/TorsionPreferences.cpp

```diff
@@ -151,12 +161,38 @@ void getExperimentalTorsions(const RDKit::ROMol &mol, CrystalFFDetails &details,
   unsigned int aid1, aid2, aid3, aid4;
   unsigned int bid2;
 
+  // exclude bonds in bridged ring systems
+  boost::dynamic_bitset<> excludedBonds(nb);
+  const RingInfo *rinfo = mol.getRingInfo();
+  const VECT_INT_VECT &bondRings = rinfo->bondRings();
+  VECT_INT_VECT_CI rii, rjj;
+  for (rii = bondRings.begin(); rii != bondRings.end(); rii++) {
```

Comment:
You should always pre-increment iterators. It can save the work of making an extra copy of the iterator.
```suggestion
  for (rii = bondRings.begin(); rii != bondRings.end(); ++rii) {
```